src/catalog/src/durable.rs

```diff
@@ -163,6 +163,8 @@ pub trait ReadOnlyDurableCatalogState: Debug + Send {
     async fn get_timestamps(&mut self) -> Result<Vec<TimelineTimestamp>, CatalogError>;
 
     /// Get all audit log events.
+    ///
+    /// Results are guaranteed to be sorted by ID.
```

Comment:
Previously, these results just happened to be sorted, but the binary schema changed that.